docs/en/developement/NewLicenseGuide.md

```diff
@@ -2,9 +2,23 @@
 
 If you have any new Jar binary package adding in you PR, you need to follow the steps below to notice license
 
-1. declared in `tools/dependencies/konw-dependencies.txt`,
-2. Add the corresponding License file under `seatunnel-dist/release-docs/licenses`, if it is a standard Apache License, it does not need to be added.
+1. declared in `tools/dependencies/konw-dependencies.txt`
```

Comment:
```suggestion
1. declared in `tools/dependencies/known-dependencies.txt`
```

Reply:
Hi @morningman, thanks for your comment. I just resolved this in another PR due to it is not strong relevant, FYI, https://github.com/apache/incubator-seatunnel/pull/1263